curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-ask-and-share-about-educational-and-professional-background/657b30e1b9f035e7e656fd01.md

```diff
@@ -0,0 +1,54 @@
+---
+id: 657b30e1b9f035e7e656fd01
+title: Task 75
+challengeType: 19
+dashedName: task-75
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE:
+Anna: I saw that tech companies appreciate innovation and care for the employees’ well-being.
+-->
+
+# --description--
+
+An `employee` is someone who works for a company or a person. When you talk about `employees' well-being`, you are talking about the health and happiness of all the people who work there. The `'s` after `employees` means that the well-being belongs to the employees. It's like `the cat's toy` (one cat) and `the cats' toy` (more than one cat). 
+
+# --question--
+
+## --text--
+
+What does Anna mean by saying “employees’ well-being."
```

Comment:
```suggestion
What does Anna mean by saying `employees’ well-being.`
```

"Smart" quotes will forever be the bane of my existence.

Reply:
```suggestion
What does Anna mean by saying `employees' well-being.`
```
Did anyone decide about curly vs straight?

Reply:
I'm fighting for straight. I hate curly quotes in my code files haha

Reply:
I'm with you on this one.

Reply:
Straightening. We can revisit later if necessary.